package.json

```diff
@@ -113,7 +113,7 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.1.4",
-    "fundamental-styles": "0.14.0",
+    "fundamental-styles": "0.16.1",
```

Comment:
Have to remove this line for now (esp. since the styles are hardcoded). Upgrading styles versions can be a big undertaking - check out this PR https://github.com/SAP/fundamental-ngx/pull/4773

Reply:
@mikerodonnell89 yeah, that's why I've noted it in the first line of the PR description. But a new version of `fundamental-styles` has a fix for Avatar component styles as well, and a new version needs for a correct Avatar Group component styles